### ISSUE TITLE
Use kueue metrics in load test

### ISSUE THREAD

**Issue by alculquicondor** [kubernetes-sigs/kueue#655]
**What would you like to be added**:

There are a number of metrics exported by kueue that could be used to measure its performance under load. We need to complete https://github.com/kubernetes-sigs/kueue/pull/481 and update README.

**Why is this needed**:

For better tracking of kueue performance

**Completion requirements**:

This enhancement requires the following artifacts:

- [ ] Design doc
- [ ] API change
- [ ] Docs update

The artifacts should be linked in subsequent comments.

**Comment by mikouaj** [comment]
@alculquicondor I've drafted Grafana dashboard that leverages metrics from Kueue for purpose of HPC on GKE workshops, do you think this might be useful?

**Comment by alculquicondor** [comment]
Yeah, they could be useful in the context of load tests but also in production.

Can grafana dashboards be shared as configuration files? Otherwise you could add a documentation page here https://github.com/kubernetes-sigs/kueue/tree/main/site/content/en/docs/tasks

Do you mind opening a separate issue for this?

**Comment by mikouaj** [comment]
FYI [here is the dashboard](https://github.com/mikouaj/gke-kueue-demo/tree/main/assets) that I have created and play with.
Sure, I can create the separate issue for this.

**Comment by alculquicondor** [comment]
cc @moficodes @alizaidis
